<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storescene</class>
 <widget class="QMainWindow" name="storescene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exchangeButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Exchange {1 square + 1 circle + 2 half square + 2 half circle} for 10 gold coins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyDeviceButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>buyDeviceButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDeliveryCenterButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>710</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>addDeliveryCenterButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSourceButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>addSourceButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="goldLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>gold num</string>
    </property>
   </widget>
   <widget class="QLabel" name="squareOreLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>square num</string>
    </property>
   </widget>
   <widget class="QLabel" name="circleOreLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>circle num</string>
    </property>
   </widget>
   <widget class="QLabel" name="halfSquareOreLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>half square num</string>
    </property>
   </widget>
   <widget class="QLabel" name="halfCircleOreLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>half circle num</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/resource/images/shop_scene.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>exchangeButton</zorder>
   <zorder>buyDeviceButton</zorder>
   <zorder>addDeliveryCenterButton</zorder>
   <zorder>addSourceButton</zorder>
   <zorder>goldLabel</zorder>
   <zorder>squareOreLabel</zorder>
   <zorder>circleOreLabel</zorder>
   <zorder>halfSquareOreLabel</zorder>
   <zorder>halfCircleOreLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
